docs/cloud-wallet.md

```diff
@@ -99,6 +99,22 @@ const document = {
 await wallet.addDocument(document);
 ```
 
+### Issuing Credentials to Cloud Wallet
+
+You can issue credentials directly to a cloud wallet using the Truvera Workspace/API. The credential will be automatically distributed to the holder's cloud wallet through the DIDComm protocol, eliminating the need for direct API calls or manual credential handling.
+
+#### Important Requirement
+
+For the DIDComm automatic distribution to work properly, the **subject ID of the credential must be set to the holder's DID** when issuing the credential. This enables the system to route the credential to the correct wallet.
+
+#### Receiving Credentials in Cloud Wallet
+
+After a credential has been issued to a holder's DID, the cloud wallet only needs to fetch and process DIDComm messages to receive it:
+
+```ts
+await messageProvider.fetchMessages();
```

Comment:
do we cover initializing the `messageProvider` somewhere else? My first question reading this would be, "how does the messageProvider know what DIDs?"

Reply:
We don't cover anything related to the messageProvider in the cloud wallet because that is not cloud wallet specific. I see it might be confusing for someone starting out with the cloud wallet w/o having wallet-sdk knowledge. I will extend the code snippet to include the initialization of the message provider.